File: portofolio personal/assets/projects/kalkulator.py
#kalkulator

print(20*"_" , )
print(20*" " , )

uinput1 = float(input("Masukan angka : "))
operator = input("Masukan operator : ")
uinput2 = float(input("Masukan angka : "))

if operator == "+" :
        print(uinput1 + uinput2)
elif operator == "-" :
        print(uinput1 - uinput2)
elif operator == "*" :
        print(uinput1 * uinput2)
elif operator == "/" :
        print(uinput1 / uinput2)

print("Akhir")

File: portofolio personal/assets/projects/kalkulator.cpp
#include <iostream>
using namespace std;

int main(){
    
    float a, b, hasil;
    char di;
    
    cout << "Kalkulator sederhana" << endl;
    cout << "masukan angka: ";
    cin >> a;
    cout << "masukan operator (+ - * /): "; 
    cin >> di;
    cout << "masukan angka: ";
    cin >> b;
    
    switch(di){
        case '+':
            hasil = (a + b);
            cout << "hasil = " << hasil << endl;
            break;
        case '-':
            hasil = (a - b);
            cout << "hasil = " << hasil << endl;
            break;
        case '*':
            hasil = (a * b);
            cout << "hasil = " << hasil << endl;
            break;
        case '/':
            if (b == 0){
                cout << "error! Tidak bisa dibagi dengan nol." << endl;
            } else {
                hasil = (a / b);
                cout << "hasil = " << hasil << endl;
            }
            break;
        default:
            cout << "operator yang anda masukan salah" << endl;
            break;
    }
    
    cout << "selesai" << endl;
}
